src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorApi.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Blazor.Razor
+{
+    // Constants for method names used in code-generation
+    // Keep these in sync with the actual definitions
+    internal static class BlazorApi
```

Comment:
I decided to group these because otherwise they are scattered across a bunch of different files.

Reply:
OK sounds good. BTW would anything bad happen if `Microsoft.AspNetCore.Blazor.Razor.Extensions` took a dependency on `Microsoft.AspNetCore.Blazor` (which is just a netstandard2.0 package with no dependencies)? If we did that then we could use a lot of `nameof` and `typeof(...).FullName` instead of duplicating. I'm guessing you probably didn't for an important reason, but just checking.